src/components/common/localization/language-dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { useLanguageStore } from "@/stores/localization/useLanguageStore";
+import {
+    DropdownMenu,
+    DropdownMenuTrigger,
+    DropdownMenuContent,
+    DropdownMenuItem,
+} from "../../ui/dropdown-menu";
+import { FaChevronDown, FaChevronUp, FaCheck } from "react-icons/fa6";
+import { useSyncLanguage } from "@/hooks/useSyncLanguage";
+
+export const LanguageDropdown = () => {
+    useSyncLanguage();

```

Comment:
add this globally

Reply:
As discussed, have removed it since we do not need localization for now.